### PR TITLE
Improve guides

### PR DESCRIPTION
**Pull request by strieflin**

### CODE REVIEW

**Review by riedervi — APPROVED** [review]
This is actually as intended. But maybe the numeration should be one (i.e. the second 1. should be a 4.)

**Review by riedervi — CHANGES_REQUESTED** [review]
I think, this change should be made at the page before, i.e. at "Setting up the clusters". If the two clusters are set up in the wrong directory, the whole setup must be done once again. So it might be convenient to clone the git repository into a desired repository as a first step. Then one can set up the clusters in this directory.

**Comment by strieflin** [comment]
> This is actually as intended. But maybe the numeration should be one (i.e. the second 1. should be a 4.)

That's a markdown feature. Any sequence of arbitrary numbers, e.g. 1., 1., 1., will be converted into a sequence of consecutive numbers.

I recommend not to proof-read the sources but the generated site. See the README file for instructions on how to do this.

**Comment by strieflin** [comment]
> I think, this change should be made at the page before, i.e. at "Setting up the clusters". If the two clusters are set up in the wrong directory, the whole setup must be done once again. So it might be convenient to clone the git repository into a desired repository as a first step. Then one can set up the clusters in this directory.

Actually, it doesn't matter what the current directory is, when you setup the kind clusters. There is no dependency whatsoever on a file or directory in that step. The only thing that matters is that you are in the right directory when performing the steps described in the "Stack Deployment" guide, because `helmfile` expects the `helmfile.d` directory in the working directory.

**Comment by riedervi** [comment]
Thank you for these explanations. Then it's ok from my side

**Review by sbckr — DISMISSED** [review]
LGTM

File: docs/getting-started/deployment.md [hunk 1]
```diff
@@ -41,6 +41,23 @@ clusters using the kind tool as described in the
     way you have to download the images only once and then reuse them across
     VCP deployments.
 
+1. Checkout out the [carbynestack repository](https://github.com/carbynestack/carbynestack)
+   and descend into to repository root directory using:
```
Comment (grevend-bosch) [inline docs/getting-started/deployment.md:45]: into & to?

**Review by grevend-bosch — CHANGES_REQUESTED** [review]

**Review by sbckr — APPROVED** [review]
:+1:

File: docs/getting-started/deployment.md (hunk 1 above)
Comment (strieflin) [reply docs/getting-started/deployment.md]: Thx. Fixed that issue.

**Pull request merged**